mypy/semanal.py

```diff
@@ -2262,7 +2262,7 @@ def infer_metaclass_and_bases_from_compat_helpers(self, defn: ClassDef) -> None:
                     break
 
         metas = {defn.metaclass, with_meta_expr, add_meta_expr} - {None}
-        if len(metas) == 0:
+        if not metas:
```

Comment:
Would prefer to keep this one as is, as it retains symmetry with the `if len(metas) > 1` condition immediately below